packages/core/test-helpers/src/helpers.ts

```diff
@@ -3,7 +3,9 @@ import { AdapterRequest, Execute } from '@chainlink/ea-bootstrap'
 
 const mockContext = {}
 
-export function assertError(statusCode: any, data: any, expectedJobId: any) {
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function assertError(statusCode: any, data: any, expectedJobId: any): void {
```

Comment:
Are these really necessary? Seems like we should be able to get types for a few of these, right?